bin/crystal

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/bin/sh
```

Comment:
I see. My comment is still valid (but could be discussed somewhere else if needed)